amazon/ionbenchmark/ion_load_dump.py

```diff
@@ -26,12 +26,17 @@ def loads(self, s):
 
     def load(self, fp):
         ion.c_ext = self._c_ext
-        return ion.load(fp, single_value=self._single_value, value_model=self.value_model)
+        it = ion.load(fp, parse_eagerly=False, single_value=False, value_model=self.value_model)
+        while True:
+            try:
+                yield next(it)
+            except StopIteration:
+                return
 
     def dumps(self, obj):
         ion.c_ext = self._c_ext
-        return ion.dumps(obj, binary=self._binary)
+        ion.dumps(obj, binary=self._binary)
```

Comment:
does this need to change too?

Reply:
Added back.

Reply:
The question was as to whether this needed `sequence_as_stream=True`